spec/board_spec.rb

```diff
@@ -1,134 +1,14 @@
-                                          "X", "O", "O",
-                                          "O", "X", "O"])
-      expect(x_first_row_win.win_for?("O")).to eq false 
-    end
-
-    it 'determines a first row win for O' do
-      o_first_row_win = Board.with_moves(["O", "O", "O",
-                                          "O", "X", "X",
-                                          "X", "O", "X"])
-
-      expect(o_first_row_win.win_for?("O")).to eq true 
-    end
-
-    it 'determines no win for X when O wins' do
-      o_first_row_win = Board.with_moves(["O", "O", "O",
-                                          "O", "X", "X",
-                                          "X", "O", "X"])
+require 'board'
 
-      expect(o_first_row_win.win_for?("X")).to eq false 
-    end
-
-    it 'determines a second row win for X' do
-      x_second_row_win = Board.with_moves(["X", "O", "O",
-                                           "X", "X", "X",
-                                           "O", "X", "O"])
-      expect(x_second_row_win.win_for?("X")).to eq true 
-    end
-
-    it 'determines a second row win for O' do
-      o_second_row_win = Board.with_moves(["O", "X", "X",
-                                           "O", "O", "O",
-                                           "X", "O", "X"])
-      expect(o_second_row_win.win_for?("O")).to eq true 
-    end
-
-    it 'determines a third row win for X' do
-      x_third_row_win= Board.with_moves(["X", "O", "O",
-                                         "O", "O", "X",
-                                         "X", "X", "X"])
-      expect(x_third_row_win.win_for?("X")).to eq true 
-    end
-
-    it 'determines a third row win for O' do
-      o_third_row_win = Board.with_moves(["O", "X", "X",
-                                          "X", "X", "O",
-                                          "O", "O", "O"])
-      expect(o_third_row_win.win_for?("O")).to eq true 
-    end
-  end
-
-  context 'column wins' do 
-    it 'determines a first column win for X' do
-      x_first_column_win = Board.with_moves(["X", "O", "X",
-                                             "X", "O", "O",
-                                             "X", "X", "O"])
-      expect(x_first_column_win.win_for?("X")).to eq true 
-    end
-
-
-    it 'determines a first column win for O' do
-      o_first_column_win = Board.with_moves(["O", "X", "O",
-                                             "O", "X", "X",
-                                             "O", "O", "X"])
-      expect(o_first_column_win.win_for?("O")).to eq true 
-    end
+describe Board do 
 
-    it 'determines a second column win for X' do
-      x_second_column_win = Board.with_moves(["O", "X", "X",
-                                              "-", "X", "O",
-                                              "-", "X", "O"])
-      expect(x_second_column_win.win_for?("X")).to eq true 
-    end
+  context 'determines board state' do
+    it 'empty board' do
```

Comment:
This test can be passed with a simple `true`... the implementation for empty? does not need to be correct until you have two or more tests